packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.html

```diff
@@ -11,19 +11,26 @@
       class="collectionSelector mb-2 d-flex align-items-center"
```

Comment:
Use this wrapper to achieve the current layout you have using bootstrap's flex related classes.